docs/integrations/amazon-aws/sqs.md

```diff
@@ -13,235 +13,221 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
-
-**Number Of Messages Received**. See the sum of the metrics number of messages received by region in the last 24 hours on a line chart.
-
-**Sent Message Size**. See the average, sum, and maximum of the metric sent message size in the last 24 hours by region on a line chart.
-
-**Number Of Messages Sent and Deleted**. See the sum of the metric number of messages sent, and the sum of the metric number of messages deleted, in the last 24 hours by region on a line chart.
-
-
-### CloudTrail Events
-
-See the details of SQS from logs including the top users, events, and queues.
-
-<img src={useBaseUrl('img/integrations/amazon-aws/SQS-CloudTrailEvents.png')} alt="ASQS-CloudTrailEvents" />
-
-**Events. **See the count and percentage of the different SQS events in the last 14 days on a pie chart.
-
-**Events Trend**. See the trend of different events in the last 14 days on a stacked column chart.
-
-**Events Location. **See the count of SQS CloudTrail events by location in the last 14 days on a world map.
-
-**Top Queues**. See the top 10 queue name by event count in the last 14 days displayed in a table.
+```Parse Expression
+json "userIdentity", "eventSource", "eventName", "awsRegion", "recipientAccountId", "requestParameters", "responseElements", "sourceIPAddress" as userIdentity, event_source, event_name, region, recipient_account_id, requestParameters, responseElements, src_ip  nodrop
+| json field=userIdentity "accountId", "type", "arn", "userName" as accountid, type, arn, username nodrop
+| json field=requestParameters "queueUrl" as queueUrlReq nodrop
+| json field=responseElements "queueUrl" as queueUrlRes nodrop
+| where event_source="sqs.amazonaws.com"
+| if(event_name="CreateQueue", queueUrlRes, queueUrlReq) as queueUrl
+| parse regex field=queueUrl "(?<queueName>[^\/]*$)"
+| if (isBlank(recipient_account_id), accountid, recipient_account_id) as accountid
+|! toLowerCase(queuename) as queuename
+| "aws/sqs" as namespace
+| fields region, namespace, queuename, accountid
+```
 
-**Top Users**. See the top 10 users by event count in the last 14 days displayed in a table.
+## Centralized AWS CloudTrail Log Collection
 
-**Events by User**. See the count of SQS events by user in the last 14 days on a bar chart.
+In case you have a centralized collection of CloudTrail logs and are ingesting them from all accounts into a single Sumo Logic CloudTrail log source, create the following **Field Extraction Rule** to map a proper AWS account(s) friendly name/alias. Create it if not already present/update it as required.
 
-**Events Details**. See the details of SQS events including the time, event name, queue name, queue URL, type, user, account ID, AWS region, source IP address, and event count, in the last 14 days, displayed in a table.
+**Rule Name**: AWS Accounts
+**Applied at**: Ingest Time
+**Scope (Specific Data**:_sourceCategory=aws/observability/cloudtrail/logs
```

Comment:
It should be '**Scope: Specific Data**:_sourceCategory=aws/observability/cloudtrail/logs'